buildconfig/config_win.py

```diff
@@ -421,15 +421,8 @@ def setup_prebuilt_sdl2(prebuilt_dir):
     DEPS.add_dummy('PORTTIME')
     DEPS.configure()
 
-    # force use of the correct freetype DLL
-    ftDep = DEPS.add('FREETYPE', 'freetype', ['SDL2_ttf-[2-9].*', 'freetype-[1-9].*'], r'(lib)?freetype[-0-9]*\.dll$',
-                     find_header=r'ft2build\.h', find_lib=r'libfreetype[-0-9]*\.lib')
-    ftDep.path = fontDep.path
-    ftDep.inc_dir = [
-        os.path.join(prebuilt_dir, 'include').replace('\\', '/')
-    ]
-    ftDep.inc_dir.append('%s/freetype2' % ftDep.inc_dir[0])
-    ftDep.found = True
+    DEPS.add('FREETYPE', 'freetype', ['freetype'], r'freetype[-0-9]*\.dll$',
+                     find_header=r'ft2build\.h', find_lib=r'freetype[-0-9]*\.lib')
```

Comment:
Why is this line needed here? I believe the same thing is done a few lines above too

Reply:
I'm not confident enough in the control flow of config_win to say definitely without further testing. The two methods appear to be run in different "modes," but I'm only familiar with the one mode for windows buildconfig. Regardless, I think it's safe to just change them both to point to the correct freetype.